inspectacle-service/src/main/java/com/kainos/inspectacle/services/checks/ReadmeCheck.java

```diff
@@ -0,0 +1,37 @@
+package com.kainos.inspectacle.services.checks;
+
+import com.kainos.inspectacle.models.inspectacle.ProjectSummary;
+import com.kainos.inspectacle.services.GitLabApiException;
+import org.gitlab.api.GitlabAPI;
+import org.gitlab.api.models.GitlabRepositoryTree;
+
+import java.io.IOException;
+import java.util.List;
+
+public class ReadmeCheck implements Check {
+    private final GitlabAPI client;
+
+    public ReadmeCheck(GitlabAPI client) {
+        this.client = client;
+    }
+
+    @Override
+    public boolean check(ProjectSummary project) throws GitLabApiException {
+        boolean containsFile = false;
+        try {
+            List<GitlabRepositoryTree> repoTree = client.getRepositoryTree(project.getGitLabProject(), "/", "master");
```

Comment:
Assuming master branch here causes problems if people have somehow opted to not create a master branch, if you leave 3rd parameter as null it uses the repos default branch...